tests/test_optimizely_config.py

```diff
@@ -1366,23 +1366,16 @@ def test_get_audiences(self):
 
         self.assertEqual(len(config.get_audiences()), len(config_service.audiences))
 
-    def test_stringify_conditions_non_matching_ids(self):
-        '''
-            Test to confirm Non-matching ID's are handled properly by stringify_conditions functions
-        '''
-
-        experiment = {'audienceConditions': ['and', ['or', '3468206642', '3988293898'], [
-            'or', '3988293899', '3468206646', '3468206647', '3468206644', '3468206643']],
-            'audienceIds': ['0'], 'forcedVariations': {}, 'id': '1323241598'}
-
-        audience_conditions = experiment.get('audienceConditions')
-
+    def get_config_and_audiences_map_for_test(self):
```

Comment:
I see it compact with this refactor. It's still difficult to follow if all cases covered. 
What about putting all these cases in a single test (like https://github.com/optimizely/swift-sdk/blob/9851fdc0fbbae32c12d2262e92957894c7d7b2f8/Tests/OptimizelyTests-DataModel/ExperimentTests.swift#L152)?

Reply:
no problem! Can definitely do that. Let me take a look.

Reply:
Added singe test case to run all variations of audienceConditions as per swift example.